src/adapters/poi_popup.js

```diff
@@ -3,29 +3,37 @@ import OsmSchedule from '../../src/adapters/osm_schedule'
 import IconManager from "./icon_manager";
 import ExtendedString from '../libs/string'
 import ApiPoi from "./poi/idunn_poi";
-
+import Device from '../libs/device'
+import SceneState from "./scene_state";
 const poiSubClass = require('../mapbox/poi_subclass')
 let popupTemplate = require('../views/popup.dot')
 const poiConfigs = require('../../config/constants.yml').pois
 
 const WAIT_BEFORE_DISPLAY = 800
 
-
 function PoiPopup() {}
 
 PoiPopup.prototype.init = function(map) {
   this.map = map
   this.popupHandle = null
   this.timeOutHandler = null
+  this.sceneState = SceneState.getSceneState()
 }
 
 PoiPopup.prototype.addListener = function(layer) {
   this.map.on('mouseenter', layer, (e) => {
-    if(isTouchEvent(e)) {
+
+    if(Device.isMobile(e)) {
+      return
+    }
+    let poi = e.features[0]
+
+    if(this.sceneState.poi === poi.properties.global_id) {
+
+
       return
```

Comment:
This check should be done in the timeout function.  
(Otherwise the popup will still be visible after a click on a poi, when the mouse doesn't move)

And remove these blank lines :)